ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -685,6 +685,14 @@ function upgrade() {
         clear_upgrade
         check_upgrade_status "${cluster_src_ver}"
     fi
+
+    conditionalUpdates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]?.release.version' | xargs)
+    if [[ "$conditionalUpdates" == *"$TARGET_VERSION"* ]]; then
```

Comment:
```suggestion
    if [[ " $conditionalUpdates " =~ " $TARGET_VERSION " ]]; then
```
That will be more accurate.

I explained it a bit in https://github.com/openshift/release/pull/49540#discussion_r1522866207